llvm-external-projects/iree-dialects/lib/Dialect/LinalgTransform/IR/StructuredTransformOpsExt.cpp

```diff
@@ -677,18 +698,25 @@ DiagnosedSilenceableFailure transform_ext::CanonicalizedSequenceOp::apply(
     LLVM_DEBUG(DBGS() << "successfully performed: " << transform << "\n");
 
     // Canonicalization may replace payload operations associated with the
-    // transform dialect handles. Post-canonicalize reassociation is fragile and
-    // may fail. To make this less likely, drop any association that are no
-    // longer necessary, i.e., if the operand is no longer used in the sequence
-    // or elsewhere or if the result is never read.
```

Comment:
Nit: you may want to fix your editor. It seems to rewrap comments that you haven't touched and that were perfectly fine by clang-format standards. Probably you have the wrapping width set at 79 cols instead of 80.